config/routes.rb

```diff
@@ -27,4 +27,10 @@
       resources :categories, only: [:index, :new, :create, :show, :edit, :update, :destroy]
     end
   
+  get 'admin/categories/:id/words', to: 'admin/categories#show'
+
+  resources :words, only: [:index, :new, :create, :show, :edit, :update, :destroy]
+  get 'admin/categories/:id/words/:id/edit', to: 'words#edit'
+  get 'admin/categories/:id/words/new', to: 'words#new'
```

Comment:
In order to make this link work like this one
http://fels-graduates-01.herokuapp.com/admin/categories/26/words/new

You will notice this pattern "admin/categories/:id/words/"
This means that you need to put "resources words" inside of "resources categories"
This is a hint
![Screen Shot 2021-04-26 at 2 31 28 PM](https://user-images.githubusercontent.com/35232348/116038525-21f8e200-a69c-11eb-8eb7-5a741bee57c2.png)

Reply:
Next, you need set foreign_key in order to set the relationship of the CATEGORY and WORD.
This is necessary because
1. When you create a word, you add a category ID in the Words Table
2. You need a CATEGORY ID because you need to display ALL WORDS in this CATEGORY
![Screen Shot 2021-04-26 at 2 37 51 PM](https://user-images.githubusercontent.com/35232348/116039295-25409d80-a69d-11eb-903c-9d983972e1df.png)
Use this as an example,
In CATEGORY NUMBER 26 (ID: 26),
It shows all the WORDS under it. That is because all the WORDS have the CATEGORY IDs.

Reply:
The goal is to have this link and this will more likely help you continue with your work, especially the WORDS create
![Screen Shot 2021-04-26 at 2 45 56 PM](https://user-images.githubusercontent.com/35232348/116040167-40f87380-a69e-11eb-8e54-69069ce4e544.png)